docs/pages/why-keystone.tsx

```diff
@@ -40,17 +40,17 @@ export default function WhyKeystonePage() {
   return (
     <Page>
       <MWrapper>
-        <Type as="h1" look="heading92">
+        <Type as="h1" look="heading92" fontSize={['4rem', '5.75rem']}>
```

Comment:
I copied these sizes from `docs/components/primitives/Type.tsx`

Not sure if there's a more Design-Token-y way of writing this.

Reply:
Arguably we should possibly do this for all the headings with this `look`?

It's defined here: https://github.com/keystonejs/keystone/blob/master/docs/components/primitives/Type.tsx#L68-L74

Or do you think this fix should be local to this page?

Reply:
I opted to use the `<IntroHeading>` component Dominik setup on other pages which has all the responsive behaviour baked in.
hope that's cool